README.md

```diff
@@ -72,7 +72,7 @@ purposes. It is by default served via webpack-dev-server.
 
 ### To start demo app
 
-`make start` will start the demo app served by webpack-dev-server
```

Comment:
Can the makefile be removed now or is it still required?

Reply:
I was confused why there's a makefile at all. Removing it is out of scope for this PR though.